mlflow/models/__init__.py

```diff
@@ -19,21 +19,23 @@
 
 import yaml
 
-
 import mlflow
 from mlflow.utils.file_utils import TempDir
 
 
 class Model(object):
-    """An MLflow Model that can support multiple model flavors."""
+    """
+    An MLflow Model that can support multiple model flavors. It is a useful class to use in
```

Comment:
Maybe be more explicit about usefulness -- "This is a developer api aimed at implementing new Model flavors."